src/components/App/index.js

```diff
@@ -56,6 +57,7 @@ const App = () => {
             <Route path={`/${t('url.blog')}`} exact component={Blog} />
             <Route path={`/${t('url.blog')}/:post`} exact component={Blog} />
             <Route path={`/${t('url.credits')}`} exact component={Credits} />
+            <Route component={NotFound} />
```

Comment:
`NotFound` component appears underneath each page of the website. Maybe using a `Switch` component would solve the problem

![instantVisio-notFound](https://user-images.githubusercontent.com/50984435/80550659-10192c00-89c1-11ea-829d-e0a26956b679.gif)

Reply:
You're right `Switch` was missing